operators/Cargo.toml

```diff
@@ -20,7 +20,7 @@ crossbeam = "0.8"
 csv = "1.1"
 float-cmp = "0.9"
 futures = "0.3"
-gdal = "0.12"
+gdal =  { git = "https://github.com/georust/gdal.git", branch = "mdarray-future" } # "0.13"
```

Comment:
add todo?

Reply:
idk, we actually should not merge before that :disappointed:

Reply:
[2f72fa4](https://github.com/geo-engine/geoengine/pull/605/commits/2f72fa434ebd52719938ee623771bf4055ec164d)